rosdep/base.yaml

```diff
@@ -3221,6 +3221,10 @@ libqglviewer-qt4:
     xenial: [libqglviewer2-qt4]
     yakkety: [libqglviewer2-qt4]
     zesty: [libqglviewer2-qt4]
+libqglviewer-qt5:
+  arch: [libqglviewer-qt5]
```

Comment:
In your notes you say this isn't found and I can only find a qt4 version by updating the search.

Reply:
Thanks for spotting. This line was by accident and I've now removed it.